docs/user-guide/content-video-toolbar.md

```diff
@@ -0,0 +1,23 @@
+# Video Content Toolbar
+**********************
+
+As soon as a video content is added, the Video Content Toolbar appears on top of the video: 
+
+<img src="../img/content-video-toolbar/c-video-toolbar.jpg" class="ms-docimage"/>
+
+Through this toolbar, the story editor is able to perform the following operation:
+
+* **Change media source** <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> accessing the [Media Editor](media-editor-window.md)
```

Comment:
* **Change media source** <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> to open the [Media Editor](media-editor-window.md) and change (or configure) the media content